### PR TITLE
bumping version dependency for puppetlabs/apt

### LINKED ISSUE #426: Fix / update modules puppetlabs/apt dependencies
## Affected Puppet, Ruby, OS and module versions/distributions
- Puppet: 4.x / 5.x
- Ruby: 2.1.5p273
- Distribution: Debian Jessie 
- Module version: 2.7.0

## How to reproduce (e.g Puppet code you use)
e.g. by installing the puppetlabs/apt version 4.5.1 
and this module's latest version with r10k:
`
mod 'puppetlabs/apt', '4.5.1'
mod 'puppet/cassandra', '2.7.0
`

`r10k puppetfile install`

or checking the dependencies on [forge](https://forge.puppet.com/puppet/cassandra/dependencies) or the [metadata.json](https://github.com/voxpupuli/puppet-cassandra/blob/master/metadata.json#L22)

## What are you seeing

`
[R10K::TaskRunner - ERROR] Task #<R10K::Task::Module::Sync:0x00000000c07178> failed while running: "puppet module --modulepath '/home/vagrant/puppet/modules' install --version=4.5.1 --ignore-dependencies puppetlabs/apt" returned with non-zero exit value #<Process::Status: pid 16274 exit 1>
`
## What behaviour did you expect instead

latest version works with the latest apt module version.
(not sure if there are changes in this module are needed to upgrade to 4.5.x)

### PR DESCRIPTION
**Pull request by paulseward**
#### Pull Request (PR) description
Updated module dependency information to allow more recent versions of the puppetlabs/apt module

#### This Pull Request (PR) fixes the following issues
Fixes #426

### CODE REVIEW

**Comment by paulseward** [comment]
Looks like travis is failing because of these errors:
```
/home/travis/.rvm/gems/ruby-2.5.1/bin/bundle:23:in `load': cannot load such file -- /home/travis/.rvm/rubies/ruby-2.5.1/lib/ruby/gems/2.5.0/gems/bundler-1.16.3/exe/bundle (LoadError)
```
That seems to be an issue with travis rather than an issue with this MR?

File: metadata.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "dependencies": [
     {
       "name": "puppetlabs-apt",
-      "version_requirement": ">= 2.0.0 < 3.0.0"
+      "version_requirement": ">= 2.0.0 < 5.0.0"
```
Comment (bastelfreak) [inline metadata.json]: please bump this to 6.0.0

**Comment by bastelfreak** [comment]
Hi @paulseward, can you please rebase and take a look at the used email address in the commit? It isn't associated with your github account.

File: metadata.json (hunk 1 above)
Comment (paulseward) [reply metadata.json]: Done!  I hadn't spotted that 5.0.0 (and now 5.0.1) had come out :)

**Comment by bastelfreak** [comment]
Thanks!

File: metadata.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     },
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 3.0.0 < 6.0.0"
```
Comment (bastelfreak) [inline metadata.json]: well, ups.

Comment (bastelfreak) [reply metadata.json]: fixed it in https://github.com/voxpupuli/puppet-cassandra/pull/430

**Comment by khaefeli** [comment]
thank you guys!

**Pull request merged**